api/server.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const cors = require('cors');
+const helmet = require('helmet');
+
+const graphqlHTTP = require('express-graphql');
+const schema = require('../schema/schema');
+
+const mongoose = require('mongoose');
+
+const options = {
+    dbName:'d9pop4okm3g0ni',
+    user:'uasvlgqfonqekw',
+    pass:'c74d17896311862815e3626e83cd555c704ab9fd482c1b3780e4961ae853ef13'
+}
+
+mongoose.connect('https://resumeker.herokuapp.com/v1/graphql', options)
```

Comment:
I think your code looks wonderful overall, but I would feel a lot better about this line and the options object stored in a .env file and passed through as a process if at all possible.

Reply:
Moved variable to .env files. Thank you for highlighting that!